CONTRIBUTING.md

```diff
@@ -18,8 +25,60 @@ A post-install action is thrown. This action first install dependencies with `le
 
 At this point you are ready to contribute.
 
+## Storybook
+
+This tool will help you display component in a page where it will be isolated (rather than integrated in a page with many components).
+
+We are using the state of the art tool: [Storybook](https://storybook.js.org/).
+
+To launch it, you can run:
+
+```sh
+npm run storybook
+```
+
+## Keeping your forked repository up-to-date
+
+Before you start developping, it is important to always keep your forked repository **up-to-date**.
+
+### Link your forked repository to the original project
+
+For this, you will add a remote which points to the original repository (this has to be done only once):
+
+```sh
+git remote add upstream https://github.com/AxaGuilDEv/react-toolkit
+```
+
+### Update your master branch
+
+Then, each time you want to update your repository with the last changes, run:
+
+```sh
+git checkout master
+git fetch upstream
+git rebase upstream/master
+```
+
 ## How to work on a component
 
+### Architecture of a component
+
+```text
+├── dist/ : contains the generated JS/CSS files (in the postinstall process).
+├── src/
+    ├── __snapshots__ : generated storybook snapshots used to test the component.
+    ├── GreatComponent.js : contains the react component code.
```

Comment:
Maybe we can add a .stories for storybook here

Reply:
Yep, I forgot after rebasing. Added one.